index.html

```diff
@@ -21,27 +21,220 @@
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                
+                
+                
+                
+                
+            </div>
+        </section>
+        <section>
+            <div class="flex-container">
+                <div id="about-title">
+                    <div id="about-box" class="font-normal">
+                        <p>About me</p>
```

Comment:
- [ ] From the figma design shown below i have used a red circle on a background image that seems to be missing from your own portfolio design. please kindly ensure you add this background image to your project so your desgin matches closely the provided figma template as shown in the figma design below :+1: :sparkles:
- figma design
![ab-i](https://github.com/Armynerh/personal_portfolio/assets/89055502/45507611-f864-4684-a85f-e83bee6183bb)
- your design
![ab-i](https://github.com/Armynerh/personal_portfolio/assets/89055502/da7d1da9-e024-46d5-9e60-2c4298b895a0)